docs/example-scenario/ai/movie-recommendations.md

```diff
@@ -1,131 +1,132 @@
-
-All the code for this scenario is available in the [Microsoft Recommenders repository][github].
-
-Follow these steps to run the [ALS quickstart notebook][notebook]:
-
-1. [Create a DSVM][dsvm-ubuntu] from the Azure portal.
-
-2. Clone the repo in the Notebooks folder:
-
-    ```shell
-    cd notebooks
-    git clone https://github.com/Microsoft/Recommenders
-    ```
-
-3. Install the conda dependencies following the steps described in the [SETUP.md][setup] file.
-
-4. In a browser, go to your jupyterlab VM and navigate to `notebooks/00_quick_start/als_pyspark_movielens.ipynb`.
-
-5. Execute the notebook.
-
-## Related resources
-
-For an in-depth guide to building and scaling a recommender service, see [Build a real-time recommendation API on Azure][ref-arch]. For tutorials and examples of recommendation systems, see [Microsoft Recommenders repository][github].
-
-[architecture]: ./media/architecture-movie-recommender.png
-[aci]: /azure/container-instances/container-instances-overview
-[aad]: /azure/active-directory-b2c/active-directory-b2c-overview
-[aks]: /azure/aks/intro-kubernetes
-[als]: https://spark.apache.org/docs/latest/ml-collaborative-filtering.html
-[autoscale]: https://docs.azuredatabricks.net/user-guide/clusters/sizing.html#autoscaling
-[blob]: /azure/storage/blobs/storage-blobs-introduction
-[clusters]: https://docs.azuredatabricks.net/user-guide/clusters/configure.html
-[cosmosdb]: /azure/cosmos-db/introduction
-[databricks]: /azure/azure-databricks/what-is-azure-databricks
-[dsvm]: /azure/machine-learning/data-science-virtual-machine/overview
-[dsvm-id]: /azure/machine-learning/data-science-virtual-machine/dsvm-common-identity
-[dsvm-ubuntu]: /azure/machine-learning/data-science-virtual-machine/dsvm-ubuntu-intro
-[free]: https://azure.microsoft.com/free/?WT.mc_id=A261C142F
-[github]: https://github.com/Microsoft/Recommenders
-[ha]: /azure/aks/container-service-quotas
-[map]: https://en.wikipedia.org/wiki/Evaluation_measures_(information_retrieval)
-[mls]: /azure/machine-learning/service/
-[n-tier]: /azure/architecture/reference-architectures/n-tier/n-tier-cassandra
-[ndcg]: https://en.wikipedia.org/wiki/Discounted_cumulative_gain
-[notebook]: https://github.com/Microsoft/Recommenders/notebooks/00_quick_start/als_pyspark_movielens.ipynb
-[ref-arch]: /azure/architecture/reference-architectures/ai/real-time-recommendation
-[regions]: https://azure.microsoft.com/en-us/global-infrastructure/services/?products=virtual-machines&regions=all
-[resiliency]: /azure/architecture/resiliency/
-[sec-docs]: /azure/security/
-[setup]: https://github.com/Microsoft/Recommenders/blob/master/SETUP.md%60
-[sla]: https://azure.microsoft.com/en-us/support/legal/sla/virtual-machines/v1_8/
-[sla-aks]: https://azure.microsoft.com/en-us/support/legal/sla/kubernetes-service/v1_0/
-[storage-security]: /azure/storage/common/storage-service-encryption
-[vm-size]: /azure/virtual-machines/virtual-machines-linux-change-vm-size
+---
```

Comment:
This file shows a large diff. Maybe need to rebase?

Reply:
The line endings were in windows format (CRLF) and were changed to Unix style encoding after being parsed by the script.  It now matches the rest of the docs, so this is preferred anyway.

Reply:
The file was changed from windows style line endings (CRLF) to Unix style.  No text was actually changed.